webclient/dokument/models.py

```diff
@@ -605,6 +605,8 @@ def get_absolute_url(self):
         """
         Metóda pro získaní absolut url.
         """
+        if "3D" in self.ident_cely:
```

Comment:
@jnihnat Tohle mi nepřijde úplně robustní podmínka. dal bych alespoň varianty, že musí začínat na `X-C-3D`, `X-M-3D`, `C-3D`, `M-3D`. Jak to může možná nastat i jindy (třeba v určitých případech u akce).

Reply:
upraveno na kontrolu identu u dokumentu